pkg/standalone-utils/contracts/BalancerInternalLib.sol

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import { IAggregatorBatchRouter } from "@balancer-labs/v3-interfaces/contracts/vault/IAggregatorBatchRouter.sol";
+import { IAggregatorRouter } from "@balancer-labs/v3-interfaces/contracts/vault/IAggregatorRouter.sol";
+import { IRouterCommon } from "@balancer-labs/v3-interfaces/contracts/vault/IRouterCommon.sol";
+import { IBatchRouter } from "@balancer-labs/v3-interfaces/contracts/vault/IBatchRouter.sol";
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+
+import "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
+
+struct Context {
+    IAggregatorRouter aggregatorRouter;
+    IAggregatorBatchRouter aggregatorBatchRouter;
+    IVault vault;
+}
+
+library BalancerInternalLib {
```

Comment:
Would put a comment block here, but do we need this layer? BalancerSwapLib just forwards here, and doesn't add anything. What is the full intended structure here? (I'm missing some context.)